cmd/api-linter/cli.go

```diff
@@ -46,7 +46,9 @@ type cli struct {
 	ListRulesFlag           bool
 }
 
-var ExitForLintFailure = errors.New("found problems during linting")
+// ErrExistForLintFailure is an error to indicate that an error
+// was found during linting.
+var ErrExitForLintFailure = errors.New("found problems during linting")
```

Comment:
Oh can you not change the name either please? That is breaking as well.

Reply:
this will conflict with the linter, so I guess we'll have to ignore this one.

I'm looking for the way to ignore this (`// nolint:all` isn't doing the trick even), but in the meantime, I just noticed that golint was deprecated: https://github.com/golangci/golangci-lint/issues/1892 and staticheck is recommended as a replacement.

Should I do that work here? My only motivation for doing it now is that nolint isn't working for me.

Reply:
I made #1084 which has everything working with staticcheck instead. or I could merge this one in (probably will not be able to fix this one) and do staticcheck after. WDYT?

Reply:
Let's just go with your staticcheck PR, if all of the changes are the same anyways. Good call on switching, thank you.